py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -400,8 +400,21 @@ def get_decompositions(
         return {**CORE_ATEN_DECOMPOSITIONS_FILTERED, **TORCH_TRT_DECOMPOSITIONS}
     else:
         # changes made here due to torch2.6 changes https://github.com/pytorch/pytorch/pull/135080
+        decomp_table = _decomp_table_to_post_autograd_aten()
+        for key in (
```

Comment:
Can these not be added here: https://github.com/pytorch/TensorRT/blob/3110d31db850016736de4dfaaec9004bcfbf4c70/py/torch_tensorrt/dynamo/lowering/_decomposition_groups.py#L166?

Reply:
It seems that `torch_disabled_decompositions` is only used when `enable_experimental_decompositions=True`.
https://github.com/pytorch/TensorRT/blob/3110d31db850016736de4dfaaec9004bcfbf4c70/py/torch_tensorrt/dynamo/lowering/_decompositions.py#L394-L400

I can move those to `torch_disabled_decompositions` but then I'll also need to do the filtering for `enable_experimental_decompositions=False`.

Reply:
Yeah the intended flow is that explicitly enabled decomps `torch_enabled_decompositions` should be run by enable_experimental_decompositions, are these ops passing that? If they are then we probably should run filtering for both cases